critiquebrainz/frontend/templates/moderators/moderators.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Moderators') }} - CritiqueBrainz{% endblock %}
+
+{% block content %}
+<div class="container">
+  <h1>{{ _('Moderators') }}</h1>
+  <div id="list" style="margin-top: 30px">
```

Comment:
Styling should go into the Less files. This includes `width` and `height` that are defined in some places below.